src/welcome.stories.mdx

```diff
@@ -6,4 +6,12 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 This is the pattern library for Cloud Four.
 
-Details coming soon.
+## Installation
```

Comment:
I'm pretty sure the answer is "no," but is there any way to re-use this content between the README (not `.mdx`) and this story? Is there a way to add a non-output comments about the desire to keep them in sync?

https://www.jamestharpe.com/code-comments-markdown/

Reply:
I don't think there's any way to reuse it, but adding a comment is a good idea. I'll do that.